test/integration/dialects/mssql/query-queue.test.js

```diff
@@ -33,5 +34,93 @@ if (dialect.match(/^mssql/)) {
         ]);
       })).not.to.be.rejected;
     });
+
+    it('requests that reject should not affect future requests', async function() {
+      const User = this.User;
+
+      await expect(this.sequelize.transaction(async t => {
+        await Promise.all([
+          expect(User.create({
+            username: new Date()
+          })).to.be.rejected,
+          expect(User.findOne({
```

Comment:
Add delay here so we are really sure this promise is executed after the first one

Reply:
Actually in this case I could just await the queries in series